phpreport-report

```diff
@@ -318,7 +318,15 @@ class DetailedReport(Report):
 
     def get_stories_for_day_and_user(self, user, date):
         tasks_for_day = self.time_period.filter_tasks(date=date, user=user)
-        all_stories = " ".join([task.text + " " + task.story for task in tasks_for_day])
+        stories = []
+        for task in tasks_for_day:
```

Comment:
I think you could write something like this:

```
def story_in_brackets(story):
  if story:
   return "[{}]".format(story)
  else:
   return ""

all_stories = " ".join([story_in_brackets(task.story) + " " + task.text for task in tasks_for_day])
```

This seems like it might be a little easier to read. What do you think?